core/src/datayoga_core/resources/schemas/connections.schema.json

```diff
@@ -12,7 +12,7 @@
             "type": {
               "description": "DB type",
               "type": "string",
-              "pattern": "^(?!redis|cassandra$)"
+              "enum": ["mysql", "postgresql"]
```

Comment:
what about cassandra and redis?

Reply:
they have their own section in the connections, here we used to consider everything that it's not redis and cassandra as `relational.write`, I decided to list the databases we actually support.